proposal/indented-syntax-improvements.md

```diff
@@ -0,0 +1,244 @@
+# Indented Syntax Improvements
+
+*([Issue](https://github.com/sass/sass/issues/216))*
+
+This proposal improves the indented syntax format, allowing multiline
+expressions, semicolons, and curly braces.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+  * [Line breaks](#line-breaks)
+  * [Semicolons](#semicolons)
+  * [SCSS-in-Sass](#scss-in-sass)
+  * [Ending statements](#ending-statements)
+    * [After a SassScript value](#after-a-sassscript-value)
+    * [After a non-enclosed list begins](#after-a-non-enclosed-list-begins)
+    * [At-rules](#at-rules)
+  * [Design Decisions](#design-decisions)
+* [Syntax](#syntax)
+  * [IndentedStatements](#indentedstatements)
+  * [Block](#block)
+  * [WhitespaceComment](#whitespacecomment)
+  * [Whitespace](#whitespace)
+
+## Background
+
+> This section is non-normative.
+
+The indented `.sass` syntax uses whitespace to end statements (linebreaks) and
+indicate nesting (indentation). While this syntax is preferred by some authors,
+it presents authoring challenges, specifically around long lists.
+
+## Summary
+
+> This section is non-normative.
+
+This proposal changes where line breaks cause statement breaks, allows the use
+of semicolons, and adds a method to opt-in to SCSS formatting to the indented
+syntax.
+
+### Line breaks
+
+In the Sass indented syntax, line breaks always end a statement, which limits
+how authors can format their code. However, the parser can tell from context
+whether a statement can end at a given point. This document proposes that line
+breaks only end statements when a statement can end, and in any other case, a
+line break is treated as continuing white space.
+
+In this example, statements can not end after `in`, inside the interpolation, or
+after the multiplication operator, so those line breaks will not end the
+statement.
+
+```sass
+@each $item in 
+  1 2 3
+    .item-#{
+      $item
+    }
+      content: $item *
+        10
```

Comment:
I'd prefer to recommend a style where statement indentation is consistently 2 spaces and continued-expression indentation is 4 spaces to contrast.

```suggestion
@each $item in 
    1 2 3
  .item-#{
    $item
  }
    content: $item *
        10
```